priv/docker.go

```diff
@@ -1,7 +1,8 @@
-package docker
+package priv
```

Comment:
Maybe use `internal` if this package is supposed to be private?

Reply:
`priv` is for "privileges" not "private". Better naming suggestions welcome. I don't want to introduce `internal` w/o taking the time to be more thoughtful about which existing packages should be in it.